.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/rubocop"]
```

Comment:
I see. You might want to make that a separate PR since it's not directly related to the RuboCop upgrade.